src/main/java/io/cryostat/messaging/MessagingModule.java

```diff
@@ -99,23 +96,20 @@ static MessagingServer provideWebSocketMessagingServer(
     @Named(WS_MAX_CONNECTIONS)
     static int provideWebSocketMaxConnections(Environment env, Logger logger) {
         try {
-            int maxConn =
+            int count =
                     Integer.parseInt(
                             env.getEnv(
                                     Variables.MAX_CONNECTIONS_ENV_VAR,
-                                    String.valueOf(DEFAULT_MAX_CONNECTIONS)));
-            if (maxConn > MAX_CONNECTIONS) {
-                logger.info("Requested maximum WebSocket connections {} is too large.", maxConn);
-                return MAX_CONNECTIONS;
+                                    String.valueOf(Integer.MAX_VALUE)));
+            if (count <= 0) {
+                logger.warn(
+                        "{} was set to {} - ignoring", Variables.MAX_CONNECTIONS_ENV_VAR, count);
```

Comment:
Where does this log line get printed? I tried running `CRYOSTAT_MAX_WS_CONNECTIONS=0 sh run.sh
`. The max connections is properly set to `Integer.MAX_VALUE` and I don't see the log message in my logs ([logs](https://gist.github.com/jan-law/75fefd0fde4db733d2eb9b479bdec4af)).

Reply:
`run.sh` doesn't handle that env var - I'll add it in.